.examples/docker-compose/with-nginx-proxy/apache/caching.php

```diff
@@ -0,0 +1,12 @@
+<?
+$CONFIG = array (
+  'memcache.local' => '\\OC\\Memcache\\APCu',
```

Comment:
@SnowMB this line is redundant, it is already part of the image [apcu.config.php](https://github.com/nextcloud/docker/blob/master/.config/apcu.config.php)